### DISCUSSION TITLE
I would like to be able to connect my pc to the pico 4 directly

### DISCUSSION THREAD

**Discussion by LKdu29** [alvr-org/ALVR#1784] [General]
Hi ! I would like to be able to connect my pc to the pico 4 directly from the ethernet adapter plug into the pico 4 via usb c to the ethernet socket of the computer. I do it through a router at my house and it works, but I don't have a router nearby when I go play at my brother's house. The Pico 4's streaming assistant manages to detect the computer since I have put internet connection sharing in the network card settings, but when I click "connect" in the headset nothing happens. If anyone knows the solution ! Thanks !

**Comment by kruemmelbande** [comment]
Instead of a usb to ethernet adapter, look into the official way of just using alvr through usb, which is a supoprted setup.
